gamemode/modules/f1menu/cl_searchbox.lua

```diff
@@ -6,15 +6,18 @@ function PANEL:Init()
     self.BaseClass.Init(self)
     self.F1Down = true
     self:SetFont("DarkRPHUD2")
-    self:SetTextColor(Color(255,255,255,255))
-    self:SetCursorColor(Color(255,255,255,255))
 
     self.lblSearch = vgui.Create("DLabel", self)
     self.lblSearch:SetFont("DarkRPHUD2")
-    self.lblSearch:SetColor(Color(200, 200, 200, 200))
     self.lblSearch:SetText(DarkRP.getPhrase("f1Search"))
     self.lblSearch:SizeToContents()
     self.lblSearch:SetPos(5)
+
+    if GAMEMODE.Config.DarkRPSkin == "DarkRP" then
+        self:SetTextColor(Color(255,255,255,255))
+        self:SetCursorColor(Color(255,255,255,255))
+        self.lblSearch:SetColor(Color(200, 200, 200, 200))
+    end
```

Comment:
I did originally have a look in `cl_dermaskin.lua`, which I found hadn't defined the colours for anything like had the default derma skin, so figured this was probably by choice and wanted to be left alone (I've noticed myself that changing the properties of things in a SKIN table away from what the default derma skin has set can cause elements not using that skin to also be affected - can make things look very odd).

The F1 menu's search box was the only thing I found when using both the default derma skin and my own custom-made skin where colour conflictions occurred with very little you could do to otherwise prevent it (changing the colour of your search boxes for *one* element is a tad extreme), and in the case of someone using the default derma theme, nothing at all.

But that's why I opted for a more local solution than a global one, because as far as I can tell, it's only this which is troublesome; but I do agree your approach to it would be better.